apps/api/infra/api.ts

```diff
@@ -181,6 +181,22 @@ export const serviceSetup = (services: {
       },
       HSN_WEB_FORM_ID: '1dimJFHLFYtnhoYEA3JxRK',
       SESSIONS_API_URL: ref((h) => `http://${h.svc(services.sessionsApi)}`),
+      AUTH_ADMIN_API_PATHS: {
+        dev: json({
+          dev: 'https://services-auth-admin-api.internal.dev01.devland.is',
+        }),
+        staging: json({
+          dev: 'https://services-auth-admin-api.internal.dev01.devland.is',
+          staging:
+            'https://services-auth-admin-api.internal.staging01.devland.is',
+        }),
+        prod: json({
+          dev: 'https://services-auth-admin-api.internal.dev01.devland.is',
+          staging:
+            'https://services-auth-admin-api.internal.staging01.devland.is',
+          prod: 'https://services-auth-admin-api.internal.innskra.island.is',
+        }),
```

Comment:
That's fine by me. Note that dev and staging use intra-account communications. So maybe something like this (if it works).

```suggestion
        dev: ref((h) => json({
          dev: `http://${h.svc(services.authAdminApi)}`,
        })),
        staging: ref((h) => json({
          dev: 'https://services-auth-admin-api.internal.dev01.devland.is',
          staging: `http://${h.svc(services.authAdminApi)}`,
        })),
        prod: json({
          dev: 'https://services-auth-admin-api.internal.dev01.devland.is',
          staging:
            'https://services-auth-admin-api.internal.staging01.devland.is',
          prod: 'https://services-auth-admin-api.internal.innskra.island.is',
        }),
```

Reply:
It worked after changing the return type of the `json` function.